.github/component_owners.yml

```
@@ -17,67 +21,101 @@ components:
     - legendecas
   packages/opentelemetry-id-generator-aws-xray:
     - carolabadeer
+  packages/opentelemetry-propagation-utils: []
+    # Unmaintained?
```

Think @open-telemetry/javascript-maintainers or some subset should probably take this

```suggestion
    - dyladan
    - pichlermarc
    - legendecas
    - blumamir
```